crates/nu-parser/src/parser.rs

```diff
@@ -2717,34 +2717,23 @@ pub fn parse_shape_name(
         b"bool" => SyntaxShape::Boolean,
         b"cell-path" => SyntaxShape::CellPath,
         b"closure" => SyntaxShape::Closure(None), //FIXME: Blocks should have known output types
-        b"cond" => SyntaxShape::RowCondition,
-        // b"custom" => SyntaxShape::Custom(Box::new(SyntaxShape::Any), SyntaxShape::Int),
         b"datetime" => SyntaxShape::DateTime,
         b"directory" => SyntaxShape::Directory,
         b"duration" => SyntaxShape::Duration,
         b"error" => SyntaxShape::Error,
-        b"expr" => SyntaxShape::Expression,
         b"float" => SyntaxShape::Float,
         b"filesize" => SyntaxShape::Filesize,
-        b"full-cell-path" => SyntaxShape::FullCellPath,
+        b"full-cell-path" => SyntaxShape::FullCellPath, // TODO: Should users distinguish cellpath?
```

Comment:
tbh, i have no idea what a *full cell path* is as opposed to a regular *cell path* :thinking: 

one can do
```nushell
def foo [x: cell-path] { $x }
```
:+1:

Reply:
I think here I have to defer to @jntrnr on what the distinction is. https://github.com/nushell/nushell/pull/8493 introduced a literal syntax so cell-paths can easily be directly assigned and stored in their `Value` form.

Reply:
I think we could remove full-cell-path also as it'd end up just being evaluated before it could be used.